README.md

```diff
@@ -1,634 +1,1828 @@
-
-:notebook: Cardano-Loans can be adapted to support multi-loan acceptance/repayments by using a Loan ID token (a state token that links the input and output for a given loan). The main issue would again be the transaction limits. While using a more efficient language can help, the [Redundant Executions](https://github.com/cardano-foundation/CIPs/pull/418) are also a problem. The full impact of these redundant executions is still being explored.
-
-
-#### Version Compatibility
-Different versions of Cardano-Loans are not compatible with each other. That is, a borrower using v1 of the protocol cannot engage in loans with a lender using v2 of the protocol. However, they may use the same keys for both protocols, which (although resulting in different addresses/beacons) allows them to maintain their pseudonymous identities across versions. 
 
-Due to the (potentially) large feature set, it may be the case that not all features can fit into a single script. If high-level language and low level plutus-core optimizations are not enough to address this, it may be necessary to split features across multiple versions of Cardano-Loans. Besides a mild inconvenience, there is no cost to doing this; credit history transcends any one version, and users may choose what version to use depending on what features they need.
+### On-Chain, Emergent Credit History
+
+Cardano-Loans automatically generates the credit history for every borrower; there is no additional
+cost for this. Every part of the loan is tracked, not just whether it was a default. For example,
+lenders can see how much of the loan a borrower paid back before they defaulted. Therefore, not all
+defaults (should) impact creditworthiness in the same way. Futhermore, when new versions of the
+protocol come out, the borrower's credit history is not lost; a borrower's credit history transcends
+the protocol's version (assuming they are using the same credential across versions).
+
+Consider what this means for the non-banked people who live in countries unable to support their own
+credit history databases due to lack of money, corruption, etc. By using Cardano-Loans, the
+non-banked can trustlessly start building their own credit histories for as little as 0.3 ADA per loan
+action. This likely results in a credit history database that is 1,000-10,000x cheaper to run than
+the credit history systems in developed nations like the United States. And despite being
+astronomically cheaper to maintain, it offers things not even the contemporary databases offer:
+
+- Identity theft is *cryptographically impossible*.
+- Credit histories are globally accessible and verifiable.
+
+Using Cardano-Loans, the non-banked can finally leap-frog passed the incompetence/corruption of
+their own nations and build up their own, (possibly) globally recognized, economic identity.
```

Comment:
```suggestion
Using Cardano-Loans, the non-banked can finally leap-frog passed the incompetence/corruption of
their own national financial systems and build up their own, (possibly) globally recognized, economic identity.
```
Permits the possibility that a nation might not be corrupt but still have a corrupt (manipulative, coercive, repressive) financial system.